lib/index.js

```diff
@@ -33,58 +33,46 @@ Index.prototype.run = function() {
     cli.up = false;
   }
 
-  var cb = function(err) {
+  var cb = function (err) {
     if (err) {
-      console.error(err)
+      console.error(err);
       return process.exit(1);
     }
     process.exit();
-  }
+  };
 
   cli.count = cli.count || Infinity;
-  if (cli.down)
-    return this.migrant.down(cli.count, cb);
+  if (cli.down) return this.migrant.down(cli.count, cb);
 
   this.migrant.up(cli.count, cb);
 };
 
-
-Index.prototype.create = function() {
-  var isCoffee = !!require.extensions['.coffee'];
-
-  var template = (isCoffee ? [
-    ''
-    , 'exports.up = (next) ->'
-    , '  next()'
-    , ''
-    , 'exports.down = (next) ->'
-    , '  next()'
-    , ''
-  ] : [
-      ''
-    , 'exports.up = function(next) {'
-    , '  next();'
-    , '};'
-    , ''
-    , 'exports.down = function(next) {'
-    , '  next();'
-    , '};'
-    , ''
-  ]).join('\n');
+Index.prototype.create = function () {
+  var template =
+    this.opts.template ||
```

Comment:
this is the main change here.

all other changes are caused by my editors `prettier` setup.